PhysicsTools/NanoAOD/python/met_cff.py

```diff
@@ -106,6 +106,32 @@
     ),
 )
 
+deepMetResolutionTuneTable = cms.EDProducer("SimpleCandidateFlatTableProducer",
+    # current deepMets are saved in slimmedMETs in MiniAOD,
+    # in the same way as chsMet/TkMET
+    src = metTable.src,
+    name = cms.string("DeepMETResolutionTune"),
+    doc = cms.string("Deep MET trained with resolution tune"),
+    singleton = cms.bool(True), # there's always exactly one MET per event
+    extension = cms.bool(False), # this is the main table for the MET
+    variables = cms.PSet(#NOTA BENE: we don't copy PTVars here!
+        pt = Var("corPt('RawDeepResolutionTune')", float, doc="raw DeepResolutionTune pt"),
+        phi = Var("corPhi('RawDeepResolutionTune')", float, doc="raw DeepResolutionTune phi"),
+    ),
+)
+
+deepMetResponseTuneTable = cms.EDProducer("SimpleCandidateFlatTableProducer",
+    src = metTable.src,
+    name = cms.string("DeepMETResponseTune"),
+    doc = cms.string("Deep MET trained with extra response tune"),
+    singleton = cms.bool(True), # there's always exactly one MET per event
+    extension = cms.bool(False), # this is the main table for the MET
+    variables = cms.PSet(#NOTA BENE: we don't copy PTVars here!
+        pt = Var("corPt('RawDeepResponseTune')", float, doc="raw DeepResponseTune pt"),
+        phi = Var("corPhi('RawDeepResponseTune')", float, doc="raw DeepResponseTune phi"),
+    ),
```

Comment:
do you need to ad variable like sumET and uncertainties ?

Reply:
currently sumEt is not calculated in the DeepMETProducer, and it is not needed in past studies we did. If it is needed for  other people, we can included this, but it requires the updates in DeepMETProducer as well. For the uncertainties, it is a bit different from pfMET and PuppiMET since we don't have 'DeepJets' clustered with the same weights here. Our approach is to define the DeepMET uncertainties from the calibration process, and calculate it at the analysis level.

For the `phi` var, maybe it's better to change the precision to 12, to be consistent with the `phi`s in PF and Puppi MET and save a little bit space?

Reply:
good point about the precision. you can reduce the precision for both pt and phi. go ahead.